import io.netty.resolver.dns.DnsNameResolverException;
public class FunctionUse8040 {
public void funcUse() {
DnsNameResolverException dnsnameresolverexception = new DnsNameResolverException();
dnsnameresolverexception.fillInStackTrace();
}
}